pkg/common/peertracker/getpid.c

```diff
@@ -0,0 +1,59 @@
+#include "getpid.h"

```

Comment:
We should include Windows.h here and not bother including getpid.h from this compilation unit (windows.h isn't needed for the function signature so there is no reason to make it available to CGO).